<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ПУСК</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledt_a">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledt_b">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledt_a_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ПУСК</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledt_b_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>211</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Равномерное распределение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>39</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Нормальное распределение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
